Test/Tests/RealtimeClientConnectionTests.swift

```diff
@@ -2919,7 +2919,7 @@ class RealtimeClientConnectionTests: XCTestCase {
                         ttlAndIdleIntervalPassedTestsClient.connection.once(.connected) { _ in
                             XCTAssertNotEqual(ttlAndIdleIntervalPassedTestsClient.connection.id, ttlAndIdleIntervalPassedTestsConnectionId)
                             channel.once(.attached) { stateChange in
-                                XCTAssertFalse(stateChange.resumed)
+                                XCTAssertNil(stateChange.reason)
```

Comment:
I don't know why they've chosen to check whether state was resumed, since they've already checked that connections were different two lines above that. But for some reason I've decided that just removing the assertion looks goofy in a way "this stopped working, so I just remove it", so I've replaced it with at least something =) Which is, I agree, not any better. So, yes, removed (`XCTAssertFalse(stateChange.resumed)` no longer the case in v2) - 0d87a66a.